src/TextForEvent.tsx

```diff
@@ -739,6 +777,10 @@ const handlers: IHandlers = {
     [EventType.RoomMessage]: textForMessageEvent,
     [EventType.Sticker]: textForMessageEvent,
     [EventType.CallInvite]: textForCallInviteEvent,
+    [POLL_START_EVENT_TYPE.name]: textForPollStartEvent,
+    [POLL_END_EVENT_TYPE.name]: textForPollEndEvent,
+    [POLL_START_EVENT_TYPE.altName]: textForPollStartEvent,
+    [POLL_END_EVENT_TYPE.altName]: textForPollEndEvent,
```

Comment:
All of these are changing ftr - would recommend holding off on merging this until the extensible events stuff lands.